### DISCUSSION TITLE
Budibase Embedded Map

### DISCUSSION THREAD

**Discussion by JayP718** [Budibase/budibase#6650] [Help] [answered]
Hi all,

I'm trying to use NYC's tileset for the map but for some reason it's not working as expected. The docs provide a url to the tileset.

[Link to NYC Map Docs ](https://maps.nyc.gov/tiles/#leaf)

I substitute the open street map tileset with the url below but all I see is gray.

https://maps{s}.nyc.gov/xyz/1.0.0/carto/basemap/{z}/{x}/{y}.jpg

Has anyone else ran into this issue?

Thanks!

**Comment by deanhannigan** [comment, marked as answer]
Hey @JayP718, thanks for getting in touch! The supplied set relates only to the New York area and nowhere beyond. In their documentation they have specifically added bounds to lock the user in the area. We don't currently support custom bounds but you can at least utilise the tileset with the the configuration below.

- **Tile URL:** https://maps.nyc.gov/xyz/1.0.0/carto/basemap/{z}/{x}/{y}.jpg
  - With the the `{s}` domain key removed.
- **Default coordinates**: 40.730610, -73.935242
  - Within the bounds of the New York area.
- **Zoom**: 50
  - Within the usable zoom levels of the supplied tileset.

If you run in to any further issues, let us know.

**Reply by JayP718** [reply]
Hi, 

That makes a lot of sense. Thanks! @deanhannigan
